app/src/hooks/useChainflipTracker.ts

```diff
@@ -0,0 +1,160 @@
+import { captureException } from '@sentry/nextjs'
+import { useCallback, useEffect, useMemo, useRef } from 'react'
+import { type Notification, NotificationSeverity } from '@/models/notification'
+import { type CompletedTransfer, type StoredTransfer, TxStatus } from '@/models/transfer'
+import { truncateAddress } from '@/utils/address'
+import { updateTransferMetrics } from '@/utils/analytics'
+import { chainflipRefund, getChainflipOngoingSwaps, getSwapStatus } from '@/utils/chainflip'
+import { getChainflipExplorerLink } from '@/utils/transfer'
+import useCompletedTransfers from './useCompletedTransfers'
+import useNotification from './useNotification'
+import useOngoingTransfers from './useOngoingTransfers'
+
+const handleOngoingSwap = async (
+  swap: StoredTransfer,
+  updateStatus: (id: string, newStatus?: string) => void,
+  updateProgress: (id: string) => void,
+  addCompletedSwap: (completedTransfer: CompletedTransfer) => void,
+  removeOngoingSwap: (id: string) => void,
+  addNotification: (notification: Omit<Notification, 'id'>) => void,
+): Promise<void> => {
+  const { sourceToken, sourceChain, destChain, destinationToken, id, uniqueTrackingId, recipient } = swap
+  if (!uniqueTrackingId) throw new Error(`Chainflip swap ${id} does not have a unique tracking id`)
+  const status = await getSwapStatus(uniqueTrackingId)
+  const { state } = status
+  const formattedRecipient = truncateAddress(recipient)
+  switch (state) {
+    case 'RECEIVING': {
+      const newStatus = `${sourceToken.symbol} funds deposited`
```

Comment:
```suggestion
      const newStatus = `${sourceToken.symbol} deposited`
```

I don't think we need "funds" here, as we don't do either in the other messages.